airbyte-integrations/connectors/source-monday/source_monday/schemas/boards.json

```diff
@@ -27,7 +27,7 @@
         "title": { "type": ["null", "string"] }
       }
     },
-    "id": { "type": ["null", "integer"] },
+    "id": { "type": ["null", "string"] },
```

Comment:
changing the field types is a breaking change. we'll need to coordinate with TCS on releasing this if it's used in Cloud. cc @YowanR